File: application/models/Agenda.php
<?php

class Application_Model_Agenda extends Application_Model_Element_Abstract {
    protected $_idField = 'ageid';
    protected $_utiid;
    protected $_interid;
    protected $_agedeb;
    protected $_agefin;
    protected $_agelieu;
    protected $_agesuppr;
    
    public function getUtiid() {
        return $this->_utiid;
    }

    public function setUtiid($utiid) {
        $this->_utiid = $utiid;
    }

    public function getInterid() {
        return $this->_interid;
    }

    public function setInterid($interid) {
        $this->_interid = $interid;
    }

    public function getAgedeb() {
        return $this->_agedeb;
    }

    public function setAgedeb($agedeb) {
        $this->_agedeb = $agedeb;
    }

    public function getAgefin() {
        return $this->_agefin;
    }

    public function setAgefin($agefin) {
        $this->_agefin = $agefin;
    }

    public function getAgelieu() {
        return $this->_agelieu;
    }

    public function setAgelieu($agelieu) {
        $this->_agelieu = $agelieu;
    }

    public function getAgesuppr() {
        return $this->_agesuppr;
    }

    public function setAgesuppr($agesuppr) {
        $this->_agesuppr = $agesuppr;
    }




}

?>

File: src/models/Agenda.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package models;

import java.io.Serializable;
import java.util.Date;
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;
import javax.persistence.Temporal;

/**
 *
 * @author overadmin
 */
@Entity
@Table(name="agenda")
public class Agenda implements Serializable{
    
    @Id
    @GeneratedValue(strategy=GenerationType.IDENTITY)
    private int cliid;
    
    @Column(name="utiid", columnDefinition="INTEGER", nullable=true)
    private int utiid ;
    
    @Column(name="ageintitule", columnDefinition="VARCHAR(254)", nullable=true)
    private String ageintitule;
    
    @Column(name="agedeb", columnDefinition="timestamp without time zone", nullable=true)
    @Temporal(javax.persistence.TemporalType.TIMESTAMP)
    private Date agedeb;
    
    @Column(name="agefin", columnDefinition="timestamp without time zone", nullable=true)
    @Temporal(javax.persistence.TemporalType.TIMESTAMP)
    private Date agefin;
    
    @Column(name="agelieu", columnDefinition="VARCHAR(50)", nullable=true)
    private String agelieu;
    
    @Column(name="agedesc", columnDefinition="VARCHAR(1000)", nullable=true)
    private String agedesc;
    
     @Column(name="ageetat", columnDefinition="BOOLEAN", nullable=true)
    private boolean ageetat;
    
    @Column(name="agesuppr", columnDefinition="BOOLEAN", nullable=true)
    private boolean agesuppr;
     
    @Column(name="ageuniqid", columnDefinition="VARCHAR(500)", nullable=true)
    private String ageuniqid ;

    public int getCliid() {
        return cliid;
    }

    public void setCliid(int cliid) {
        this.cliid = cliid;
    }

    public int getUtiid() {
        return utiid;
    }

    public void setUtiid(int utiid) {
        this.utiid = utiid;
    }

    public String getAgeintitule() {
        return ageintitule;
    }

    public void setAgeintitule(String ageintitule) {
        this.ageintitule = ageintitule;
    }

    public Date getAgedeb() {
        return agedeb;
    }

    public void setAgedeb(Date agedeb) {
        this.agedeb = agedeb;
    }

    public Date getAgefin() {
        return agefin;
    }

    public void setAgefin(Date agefin) {
        this.agefin = agefin;
    }

    public String getAgelieu() {
        return agelieu;
    }

    public void setAgelieu(String agelieu) {
        this.agelieu = agelieu;
    }

    public String getAgedesc() {
        return agedesc;
    }

    public void setAgedesc(String agedesc) {
        this.agedesc = agedesc;
    }

    public boolean isAgeetat() {
        return ageetat;
    }

    public void setAgeetat(boolean ageetat) {
        this.ageetat = ageetat;
    }

    public boolean isAgesuppr() {
        return agesuppr;
    }

    public void setAgesuppr(boolean agesuppr) {
        this.agesuppr = agesuppr;
    }

    public String getAgeuniqid() {
        return ageuniqid;
    }

    public void setAgeuniqid(String ageuniqid) {
        this.ageuniqid = ageuniqid;
    }   
}
